### ISSUE TITLE
Vendure Create Fails on Windows

### ISSUE THREAD

**Issue by drkNsubuga** [vendure-ecommerce/vendure#1428]
**The Issue**
'create' is not recognized as an internal or external command, operable program or batch file.

**What I tried**
```
npx @vendure/create my-app --log-level verbose
npx init @vendure my-app
```
**Environment**
 - Nodejs version 17.5
 - NPM version 8.5

**Comment by michaelbromley** [comment]
Hi there,

Can you try with Node v16? Vendure doesn't officially support odd-numbered Node releases. Perhaps it has some incompatibility. Although the error message is rather confusing...

**Comment by drkNsubuga** [comment]
Hello Michael,
Thank you so much for the feedback. I forgot to mention I'm using Windows10 x64. I managed to resolve the issue by first running ```npm i -g @vendure/create```.  
However, I'm now encountering an issue similar to  #293. This is part of the log:

```
 npm/  Installing dependencies
ERR!     → Installing @vendure/core, @vendure/email-plugin, @vendure/asset-server-plugin, @vendure/admin-ui-plugin, better-sqlite3path
command   C:\WINDOWS\system32\cmd.exe /d /s /c prebuild-install || npm run build-release
- Installing dependenciesnpm
     → Installing @vendure/core, @vendure/email-plugin, @vendure/asset-server-plugin, @vendure/admin-ui-plugin, better-sqlite3ERR!
 > better-sqlite3@7.5.0 build-release    Generating app scaffold

    Initializing servernpm
 ERR! > node-gyp rebuild --release
npm ERR!

ERR!         ^\
 Installing dependencies
npm     → Installing @vendure/core, @vendure/email-plugin, @vendure/asset-server-plugin, @vendure/admin-ui-plugin, better-sqlite3ERR!
    Generating app scaffold

       at Object.symlinkSync (node:fs:1655:3)
|npm Installing dependencies
     ERR!→ Installing @vendure/core, @vendure/email-plugin, @vendure/asset-server-plugin, @vendure/admin-ui-plugin, better-sqlite3
oper\Projects\DRK\schoolshop\site\vendure\node_modules\better-sqlite3\deps\symlink.js:18:5)    Generating app scaffold

    Initializing servernpm
 ERR!       at Module._compile (node:internal/modules/cjs/loader:1097:14)
npm ERR!       at Object.Module._extensions..js (node:internal/modules/cjs/loader:1151:10)
npm ERR!       at Module.load (node:internal/modules/cjs/loader:975:32)
npm ERR!       at Function.Module._load (node:internal/modules/cjs/loader:822:12)
npm ERR!       at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:77:12)
npm ERR!       at node:internal/main/run_main_module:17:47 {
npm ERR!     errno: -4048,
npm ERR!     syscall: 'symlink',
npm ERR!     code: 'EPERM',
npm ERR!     path: 'E:\\Developer\\Projects\\DRK\\schoolshop\\site\\vendure\\node_modules\\better-sqlite3\\deps\\sqlite3\\sqlite3.c',
npm ERR!     dest: 'E:\\Developer\\Projects\\DRK\\schoolshop\\site\\vendure\\node_modules\\better-sqlite3\\build\\Release\\obj\\global_intermediate\\sqlite3\\sqlite3.c'npm ERR!   }
npm ERR!
npm ERR!   Node.js v17.5.0
npm ERR! C:\Program Files (x86)\Microsoft Visual Studio\2019\Enterprise\MSBuild\Microsoft\VC\v160\Microsoft.CppCommon.targets(241,5): error MSB8066: Custom build for '..\..\deps\sqlite3\sqlite3.c' exited with code 1. [{project_path}\node_modules\better-sqlite3\build\deps\locate_sqlite3.vcxproj]
npm ERR! gyp ERR! build error
npm ERR! gyp ERR! stack Error: `C:\Program Files (x86)\Microsoft Visual Studio\2019\Enterprise\MSBuild\Current\Bin\MSBuild.exe` failed with exit code: 1
npm ERR! gyp ERR! stack     at ChildProcess.onExit (C:\Program Files\nodejs\node_modules\npm\node_modules\node-gyp\lib\build.js:194:23)
npm ERR! gyp ERR! stack     at ChildProcess.emit (node:events:526:28)
npm ERR! gyp ERR! stack     at Process.ChildProcess._handle.onexit (node:internal/child_process:291:12)
npm ERR! gyp ERR! System Windows_NT 10.0.19044
npm ERR! gyp ERR! command "C:\\Program Files\\nodejs\\node.exe" "C:\\Program Files\\nodejs\\node_modules\\npm\\node_modules\\node-gyp\\bin\\node-gyp.js" "rebuild" "--release"
npm ERR! gyp ERR! cwd {project_path}\node_modules\better-sqlite3
npm ERR! gyp ERR! node -v v17.5.0
npm ERR! gyp ERR! node-gyp -v v8.4.1
  × Installing dependencies
    → An error occurred when installing dependencies.
    Generating app scaffold
    Initializing server
{"message":"An error occurred when installing dependencies.","command":"npm install --save --save-exact --loglevel error @vendure/core @vendure/email-plugin @vendure/asset-server-plugin @vendure/admin-ui-plugin better-sqlite3","context":{}}
```

**Comment by michaelbromley** [comment]
OK, we're getting closer 😄 
Did you install the windows build tools as mentioned in the [getting started guide](https://www.vendure.io/docs/getting-started/)?

![image](https://user-images.githubusercontent.com/6275952/154273611-de40686e-e716-4543-a5d3-cc6bc3a6b3cd.png)

**Comment by drkNsubuga** [comment]
Oh...I tried but ran out of patience when it failed.  Let me try again. We'll get back once it's done. Thanks.

**Comment by drkNsubuga** [comment]
Hooray...it worked!

**Comment by michaelbromley** [comment]
Glad to hear it!
